libmscore/stafftype.cpp

```diff
@@ -240,6 +242,7 @@ void StaffType::write(XmlWriter& xml) const
             xml.tag("stemsDown",        _stemsDown);
             xml.tag("stemsThrough",     _stemsThrough);
             xml.tag("upsideDown",       _upsideDown);
+            xml.tag("showTabFingering",       _showTabFingering);
```

Comment:
OK. In 2.1 I always have to change the CMakeLists.txt: delete -PIE. 
As I now wanted to checkout master, it refused to do so, because of this changed file. Should I stash it, or what can I do best (checkout would also be a possibility, but I will than have to change it again ...)?